scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -727,6 +728,7 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
              * Revert item to given version
              */
             $scope.revert = function(version) {
+                $scope.refreshTrigger = !$scope.refreshTrigger;
```

Comment:
I think it's better to use an integer which increments when you need to update the state. When booleans are used I was intuitively thinking that update happens after boolean is set to `true` which is not the case.